### PR TITLE
updated soar doc link

### LINKED ISSUE #406: Link to sunpy-soar README is broken
On the affiliated package page, the link to an anchor in the sunpy-soar README is broken: 

```
(      affiliated: line   94) broken    https://github.com/sunpy/sunpy-soar#readme - Anchor 'readme' not found
```

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #406

### CODE REVIEW

**Comment by wtbarnes** [comment]
Another one is broken: https://github.com/sunpy/sunpy.org/actions/runs/8423853246/job/23066463590?pr=407#step:10:325

**Comment by nabobalis** [comment]
Can't believe you are making me fix links from 2013! :P

File: posts/2013/2013-07-21-sunpy-at-scipy.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ This diversity in people's backgrounds and interest lead to the most diverse, in
 
 The conference this year consisted of two days of tutorials, two days of talks and two days of sprints.
 The tutorials covered different levels and topics ranging from introductions to IPython, matplotlib and scikit-learn to advanced talks on Cython and NumPy.
-Like with all the sessions at SciPy there are very good quality videos available online of the `tutorials <https://conference.scipy.org/scipy2013/tutorials_schedule.php>`_.
+Like with all the sessions at SciPy there are very good quality videos available online of the `tutorials <http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2013/mumford.html>`__.
```
Comment (nabobalis) [inline posts/2013/2013-07-21-sunpy-at-scipy.rst:19]: This feels like they will fix it in the future? But currently this is the url I get.

Comment (nabobalis) [inline posts/2013/2013-07-21-sunpy-at-scipy.rst:19]:
```suggestion
Like with all the sessions at SciPy there are very good quality videos available online of the `tutorials <http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2013/>`__.
```

File: posts/2013/2013-07-21-sunpy-at-scipy.rst [hunk 2]
```diff
@@ -16,7 +16,7 @@ This diversity in people's backgrounds and interest lead to the most diverse, in
 
 The conference this year consisted of two days of tutorials, two days of talks and two days of sprints.
 The tutorials covered different levels and topics ranging from introductions to IPython, matplotlib and scikit-learn to advanced talks on Cython and NumPy.
-Like with all the sessions at SciPy there are very good quality videos available online of the `tutorials <https://conference.scipy.org/scipy2013/tutorials_schedule.php>`_.
+Like with all the sessions at SciPy there are very good quality videos available online of the `tutorials <http://conference.scipy.org.s3-website-us-east-1.amazonaws.com/proceedings/scipy2013/>`__.
```
Comment (nabobalis) [inline posts/2013/2013-07-21-sunpy-at-scipy.rst:19]: This is all I can find now.

**Review by wtbarnes — APPROVED** [review]

**Pull request merged**